kura/examples/org.eclipse.kura.example.camel.publisher/src/org/eclipse/kura/example/camel/publisher/ExamplePublisher.java

```diff
@@ -117,16 +117,16 @@ protected PublisherConfiguration parseConfiguration(final Map<String, Object> pr
 
         result.put("intValue",
                 round(makeSine(configuration.getAmpInt(), configuration.getOffsetInt(), configuration.getPeriodInt())));
-        result.put("doubleValue",
-                makeSine(configuration.getAmpDouble(), configuration.getOffsetDouble(), configuration.getPeriodDouble()));
+        result.put("doubleValue", makeSine(configuration.getAmpDouble(), configuration.getOffsetDouble(),
+                configuration.getPeriodDouble()));
 
         // return result
 
         return result;
     }
 
     private static double makeSine(final double amp, final double offset, final double period) {
-        final double freq = 1.0 / period * Math.PI;
+        final double freq = 1.0 / period * Math.PI * 2.0;
```

Comment:
This could be written as:
`final double freq = 2.0 / period * Math.PI;`

Or even better as:
`final double freq = 2.0 * Math.PI / period;`